Rmarkdown/Multiple_Ascending_Dose_PD_binary.Rmd

```diff
@@ -148,7 +149,7 @@ When overlaying individual binomial data, it is often helpful to stagger the dot
   gg <- gg + labs(y = pd_label)  
   gg <- gg + facet_grid(~TRTACT_low2high)
   gg <- gg + xgx_annotate_status(status)
-  #gg <- gg + xgx_annotate_filenames(dirs)
```

Comment:
@margoal1 - Is the plan to comment out xgx_annotate_filenames() everywhere?  I wasn't aware we decided to do that, so I just thought I'd check in.  I guess maybe we should decide if we keep it or not because I saw some pages have it and some don't.

Reply:
The lines that are commented out were already commented out previously. It seems that the lines are only NOT commented on the PK pages, but they are commented out on the PD & PKPD pages. I don't know when that was done, or if they were ever not commented out on these pages. 

In my opinion, I have always thought the file names looked a bit odd on xGx (even though I use them on all my official plots). They used to look a lot worse back when they had all that temp directory stuff in it. Now they do look better, but still a bit odd because it's generic text ("Parent_directory", "example.R"). What if we made these not generic, e.g. reflecting the website layout somehow? For example "Parent_directory" could be "Dose-PK/Exposure" and "example.R" could be "Single_Ascending_Dose_PK.Rmd"?